SETUP.md

```diff
@@ -0,0 +1,300 @@
+# Setup
+
+This setup guide uses Docker. Our Docker container supports building the Zephyr applications, as well as flashing and debugging the boards.
+
+The supported platforms are Windows/x86_64, macOS/aarch64, Linux/x86_64. Other platforms are likely to work, but installing pyOCD might need extra steps.
+
+>> **On Windows, this guide uses Powershell as a host terminal. Please don't run the commands in WSL or Git Bash or cmd.**
+
+If you use Linux/WSL and want to proceed with the manual setup instead, please take a look at `MANUAL_SETUP.md` for more details.
+
+## Installation
+
+### 1. Install Git, Python
+
+Git is used to track and manage changes of our code. Python is used in the pyOCD debugger as well as few other scripts and tools.
+
+<details><summary>Windows</summary>
+
+Make sure `python` and `git` are installed **and are available from Powershell**.
+
+You can use [this](https://www.python.org/downloads/) link to install Python 3. Select option 'Add Python to PATH' in the installer.
+
+To install Git, use [this](https://git-scm.com/downloads/win) link. Git will be added to PATH by default.
```

Comment:
winget comes with Windows without extra installation IIRC
maybe worth mentioning winget
(just a suggestion, it's good as is)